packages/maskbook/src/plugins/ITO/SNSAdaptor/hooks/useSpaceStationCampaignInfo.ts

```diff
@@ -0,0 +1,6 @@
+import { useAsyncRetry } from 'react-use'
+import { getCampaignInfo } from '../../Worker/apis/spaceStationUUPS'
+
+export function useSpaceStationCampaignInfo() {
+    return useAsyncRetry(getCampaignInfo, [])
```

Comment:
Are you sure calling those APIs without using RPC?

Reply:
Hi, this API should return same data no matter which chain you are on.

Reply:
I don't think so. Calling them directly in SNS might cause CSP limitation.

Reply:
The API provider is our project cooperator. We are at the same Wechat group.

Reply:
The CSP limitation is set by the SNS network, not the API provider.